ansible/roles/disable_hyperthreading/files/disableHT.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Disable Hyper-Threading at Runtime
+Documentation=https://access.redhat.com/security/vulnerabilities/L1TF https://access.redhat.com/solutions/352663
+Requires=atomic-openshift-node.service
+Before=atomic-openshift-node.service
+
+[Service]
+User=root
+Type=oneshot
```

Comment:
Reading the docs, it sounds like we might want to also add ``nosmt`` to the kernel boot parameters so that the change is not only applied at runtime but persisted through reboots. Should we do both ?

Reply:
No, discussed in email thread.  We have a single AMI for master, infra, and compute for all styles of nodes.  We want to keep it this way.  Only a subset will have hyper-threading disabled.